src/pages/articles/bootstrap/bootstrap-get-started/index.md

```diff
@@ -2,14 +2,30 @@
 title: Bootstrap Get Started
 ---
 ## Bootstrap Get Started
+When using Bootstrap for the first time it can appear to be a confusing process, but using Bootstrap is actually quite simple. Using Bootstrap makes it easy to design a fully responsive website and is a framework worth learning.
 
-This is a stub. [Help our community expand it](https://github.com/freecodecamp/guides/tree/master/src/pages/articles/bootstrap/bootstrap-get-started/index.md).
+#### What is a responsive website:
+A responsive website is a website that resizes and rearranges that items on the page depending on the size of your browser. With a responsive website if you resize your browser you can see the changes occur in real time. Bootstrap makes your website responsive for you.
 
-[This quick style guide will help ensure your pull request gets accepted](https://github.com/freecodecamp/guides/blob/master/README.md).
+#### How do I add Bootstrap to my page?
+Adding bootstrap to your page is quite simple, you simple add the following to the `<head>` section of your code.
+```
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+<!-- Optional theme -->
```

Comment:
Themes are not required to add Bootstrap to a page.

Maybe this could be a separate article?